packages/@okta/vuepress-site/docs/guides/pwd-optional-sign-in-email/main/react/integrationsteps.md

```diff
@@ -32,7 +32,11 @@ Display &mdash; either statically or dynamically using `IdxTransaction` &mdash;
 
 <div class="half border">
 
-![Screenshot showing a sign-in page with a username field and a Next button.](/img/pwd-optional/pwd-optional-sign-in-page.png)
+![A sign-in page with a username field and a Next button.](/img/pwd-optional/pwd-optional-sign-in-page.png)
+
+<!--
+Source image: https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%93%8A-Updated-Diagrams-for-Dev-Docs?node-id=1978%3A1743 pwd-optional-sign-in-page
```

Comment:
Image (https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%93%8A-Updated-Diagrams-for-Dev-Docs?node-id=1978%3A1743) and image (https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%93%8A-Updated-Diagrams-for-Dev-Docs?node-id=1975%3A2792) can be only one image.

Reply:
Technically they are two different source images on the Figma board at the moment hence different links.
See note above. I will "refactor" all the wireframe diagrams separately into single images including these ones and reset the metadata for them at that time.